src/table/TableWrapper.jsx

```diff
@@ -73,10 +75,44 @@ export default function TableWrapper(props) {
     };
   }, []);
 
+  useEffect(() => {
+    if (focusedCellCoordsState[0] < 2) {
+      tableSection.current.scrollTo({
+        top: 0,
+        behavior: 'smooth',
+      });
+    } else if (isMovingTop) {
+      setIsMovingTop(false);
+
+      const rowElements = rootElement.getElementsByClassName('sn-table-row');
+      const [x, y] = focusedCellCoordsState;
+      const cell = rowElements[x]?.getElementsByClassName('sn-table-cell')[y];
+
+      if (cell.offsetTop - cell.offsetHeight * 2 < tableSection.current.scrollTop) {
+        tableSection.current.scrollTo({
+          top: tableSection.current.scrollTop - cell.offsetHeight || 0,
```

Comment:
why `|| 0`? Could `tableSection.current.scrollTop - cell.offsetHeight` become NaN or something?

Reply:
Good catch, 
My intention here was about the calculated value could get below 0 and i was preventing it. i figured out that the compassion is wrong here since negative value when converting to boolean is still a number and would give me true.

I need to change this a bit like check if value actually getting below 0, then fallback to 0 instead.

Reply:
I'll fix it asap